### PR TITLE
Add sensor name

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/EGFDCPSp/282-all-tabs-cards-missing-sensor-package-name-wrong-sensor-package-name

### CODE REVIEW

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 1]
```diff
@@ -66,8 +66,27 @@ private extension SessionHeaderView {
                     }
                 }
             }
-            Text("\(session.type?.description ?? SessionType.unknown("").description): \(session.deviceType?.description ?? "")")
-                .font(Font.moderate(size: 13, weight: .regular))
+            // As long as we get session.deviceType = nil we should handle somehow showing those devices which were used to record
+            // [|(-)   (-)|]    |-------------------|
+            //  |   ___   |  -- | You, do something |
+            //  |_________|     |-------------------|
+            // so the idea at leat for now is this below
```
Comment (smialko) [inline AirCasting/SessionViews/SessionHeaderView.swift:73]: Monkey 💙
I'm voting to leave it and add an additional warning here (it'd be just a 43rd warning, no big deal xD).

Comment (smialko) [reply AirCasting/SessionViews/SessionHeaderView.swift:73]: uh, i see the warning, good job ;D

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 2]
```diff
@@ -66,8 +66,27 @@ private extension SessionHeaderView {
...
+                Text("\(session.type?.description ?? SessionType.unknown("").description): AirBeam3")
+                    .font(Font.moderate(size: 13, weight: .regular))
+            } else if session.type?.description == "Mobile" {
+                if session.allStreams!.count > 1 {
+                    Text("\(session.type?.description ?? SessionType.unknown("").description): AirBeam3")
+                        .font(Font.moderate(size: 13, weight: .regular))
+                } else {
+                    Text("\(session.type?.description ?? SessionType.unknown("").description): Phone Mic")
+                        .font(Font.moderate(size: 13, weight: .regular))
+                }
```
Comment (smialko) [inline AirCasting/SessionViews/SessionHeaderView.swift:85]: I think this if could be simpler. Assuming that in MVP we use either AB3 or mic we could do `if session.deviceType == .MIC` and return "Phone's mic" and else return "AirBeam3" - as simple as that ;D

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 3]
```diff
@@ -66,8 +66,27 @@ private extension SessionHeaderView {
...
+            //  |   ___   |  -- | You, do something |
+            //  |_________|     |-------------------|
+            // so the idea at leat for now is this below
+            #warning("Fix - do it better")
+            if session.type?.description == "Fixed" {
+                Text("\(session.type?.description ?? SessionType.unknown("").description): AirBeam3")
+                    .font(Font.moderate(size: 13, weight: .regular))
+            } else if session.type?.description == "Mobile" {
+                if session.allStreams!.count > 1 {
+                    Text("\(session.type?.description ?? SessionType.unknown("").description): AirBeam3")
```
Comment (smialko) [inline AirCasting/SessionViews/SessionHeaderView.swift:80]: maybe move "AirBeam3" to Strings?

**Review by smialko — COMMENTED** [review]
:D love the monkey

**Review by pawelgil — CHANGES_REQUESTED** [review]
Let's be more explicit in the warning

File: AirCasting/SessionViews/SessionHeaderView.swift (hunk 2 above)
Comment (jankrzempek) [reply AirCasting/SessionViews/SessionHeaderView.swift:85]: Wow. Haven't thought. cool!

Comment (jankrzempek) [reply AirCasting/SessionViews/SessionHeaderView.swift:85]: I know now, it is not working sadly because we get always nil ad .deviceType

**Pull request merged**